test/integration/rhel-7.6/symvers-disagreement-FAIL.patch

```diff
@@ -0,0 +1,39 @@
+From bcb86fa4e9c31379a9e2716eae29cd53ccca064f Mon Sep 17 00:00:00 2001
+From: Julien Thierry <jthierry@redhat.com>
+Date: Wed, 6 May 2020 14:30:57 +0100
+Subject: [PATCH] Test patch exported
```

Comment:
Thanks for adding these!  To add the finishing touches, it would be great if we included a short description of how/what these tests are trying to do.  I know I will probably forget in a few weeks :) 

The patch looks very similar to the one you were using when you first hit this problem.  Maybe we can pull some text from those earlier email exchanges?  Let me know if you would like help describing how this caused the CRC change (I didn't fully understand it until digging into the kabi-dw tool.)